### PR TITLE
Polish Fedora setup guides

### PR DESCRIPTION
**Pull request by seisman**
Some improvements for the Fefora setup guides.

I'll update the other guides in the similar way after approval.

### CODE REVIEW

File: source/computer/fedora-setup.rst [hunk 1]
```diff
@@ -11,13 +11,6 @@ Fedora 配置指南
    本节内容适用于 **Fedora 33 Workstation**\，不一定适用于其他 Fedora 版本。
    建议用户总是选择 Fedora 最新版本。如遇到问题，欢迎反馈。
 
-.. note::
-
-   本节大部分软件都通过命令行安装。在桌面或菜单栏中找到并点击 “Terminal” 图标以启动终端，
-   然后在终端中输入命令并按下 :kbd:`Enter` 键即可执行该命令。
-
-----
-
```
Comment (seisman) [inline source/computer/fedora-setup.rst:20]: The note makes no sense until the system is installed, so I move the note after installing this  OS.

File: source/computer/fedora-setup.rst [hunk 2]
```diff
@@ -176,7 +171,7 @@ Intel 软件开发工具包
 ^^^^^^^^^^^^^^^^^^^^
 
 `Intel oneAPI Toolkits <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
-是英特尔最新的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc`` 和 ``ifort`` 命令）。
+是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc``\，``i++`` 和 ``ifort`` 命令）。
```
Comment (seisman) [inline source/computer/fedora-setup.rst:174]: Please check if it provides the `i++` command as a C++ compiler.

File: source/computer/fedora-setup.rst [hunk 3]
```diff
@@ -32,7 +25,7 @@ Fedora 配置指南
 访问 `Fedora 官网 <https://getfedora.org/>`__ 并下载 Fedora Workstation 镜像，
```
Comment (core-man) [inline source/computer/fedora-setup.rst:25]: `Fedora Workstation 镜像文件`

File: source/computer/fedora-setup.rst (hunk 2 above)
Comment (core-man) [inline source/computer/fedora-setup.rst:174]:
```suggestion
是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc``\ ，``i++`` 和 ``ifort`` 命令）。
```

Comment (core-man) [reply source/computer/fedora-setup.rst:174]: I don't see the `i++` command although I have installed the OneAPI base and HPC HPC toolkits.

Comment (core-man) [reply source/computer/fedora-setup.rst:174]: no, see https://github.com/seismo-learn/seismology101/pull/296#discussion_r614501547.

File: source/computer/fedora-setup.rst (hunk 3 above)
Comment (seisman) [inline source/computer/fedora-setup.rst:25]:
```suggestion
访问 `Fedora 官网 <https://getfedora.org/>`__ 并下载 Fedora Workstation 镜像文件，
```

File: source/computer/fedora-setup.rst (hunk 2 above)
Comment (seisman) [reply source/computer/fedora-setup.rst:174]: Hmm, so `icc` can compile C and C++ files?

Comment (core-man) [reply source/computer/fedora-setup.rst:174]: I am testing it.

Comment (seisman) [reply source/computer/fedora-setup.rst:174]: https://scc.ustc.edu.cn/zlsc/sugon/intel/compiler_c/main_cls/bldaps_cls/common/bldaps_invoke_cl_comm.htm

It seems it provides icc and icpc, and both are the same.

Comment (core-man) [reply source/computer/fedora-setup.rst:174]: I can use both `i++` and `icpc` to complile the hello-world.c++

```C++
 #include <iostream>
 
 int main() {
      std::cout << "Hello World!";
      return 0;
 }
```
```

Comment (seisman) [reply source/computer/fedora-setup.rst:174]: Did you mean `icc` or `icc`?

Comment (core-man) [reply source/computer/fedora-setup.rst:174]: I don't have `i++` and I think Intel OneAPI don't have such command.

Comment (seisman) [reply source/computer/fedora-setup.rst:174]: OK. I think we should only mention `icc` and `ifort`. I believe most C++ users are not our target audiences.

Comment (seisman) [inline source/computer/fedora-setup.rst:174]:
```suggestion
是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc`` 和 ``ifort`` 命令）。
```

Comment (core-man) [reply source/computer/fedora-setup.rst:174]: I think so.

Comment (core-man) [reply source/computer/fedora-setup.rst]: See https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html. I installed both base and HPC toolkits. So I should have C and C++.

**Review by core-man — APPROVED** [review]

**Pull request merged**